pages/community/precice-workshop-2022.md

```diff
@@ -0,0 +1,68 @@
+---
+title: preCICE workshop 2022
+keywords: precice workshop 2022, event, events, workshop
+summary:
+permalink: precice-workshop-2022.html
+toc: false
+redirect_from: /preCICE2022/
+---
+
+<img class="img-responsive center-block" src="images/events/precice2022.svg" alt="preCICE Workshop banner" style="max-width: 500px; margin:auto;">
+
+The 3rd preCICE Workshop will be held virtually through [SimTech](https://www.simtech.uni-stuttgart.de/) and the [University of Stuttgart](https://www.uni-stuttgart.de/en/), from February 21-24, 2022. The workshop is a coming together of the preCICE community to share ideas, experiences and knowledge about using preCICE, and to learn from others in the process.
+
+[Registration](https://precice.org/precice-workshop-2022.html#registration) is now open. We look forward to welcoming back previous participants, and introducing more members to the preCICE community.
+
+Due to the uncertain times we are experiencing with the ongoing COVID-19 pandemic and the great success of the virtual workshop in February 2021, we are planning to host the workshop virtually again.
+
+If you have any questions regarding the workshop, please contact us at [`precice2022@ipvs.uni-stuttgart.de`](mailto:precice2022@ipvs.uni-stuttgart.de).
+
+## Program
+
+Coming soon.
```

Comment:
When advertising it would be good to have sth preliminary here already. Not all participants will join all days. So, for them, it is important to see what is planned on which day.

Reply:
I added something. Any comment?